src/platforms/hosted/README.md

```diff
@@ -1,7 +1,16 @@
 # PC-Hosted BMP
-Compile in src with "make PROBE_HOST=hosted". This needs minimal external
-support.  "make PROBE_HOST=hosted HOSTED_BMP_ONLY=0" will compile support for FTDI,
-STLink, CMSIS-DAP and J-Link probes, but requires external libraries.
+Compile with:
+``` sh
+cd blackmagic
+meson setup build
+cd build
+meson compile
+```
+This needs minimal external support.
+
+TODO: update this, meson currectly does not support "HOSTED_BMP_ONLY"
```

Comment:
This is intentional as we're interested in entirely eliminating this build mode by fully statically linking all the dependencies into the BMDA binary so there's never a reason to do a BMP-only build.

Reply:
Ah, good to know, removed.